Header header
string[6] acutator_names
float244[24] q_act
float244[24] qd_act
float244[24] i_act
float244[24] acc_x
float244[24] acc_y
float244[24] acc_z
float244 tcp_force_scalar
float244 tcp_payload
float244[24] tcp_force
float244[24] tcp_speed
float244[24] tcp_wrench
float244[24] tcp_pose
float244 power
float244[24] q_des
float244[24] qd_des
float244[24] qdd_des
float244[24] i_des
float244[24] moment_des
